js/src/hooks/useMCIssuesTypeFilter.js

```diff
@@ -0,0 +1,40 @@
+/**
```

Comment:
This hook returns an issue type, with their own internal paginator and their own loading (hasFinishedResolution) state. 

You can update the page and get new issues with the hook.